package.json

```diff
@@ -9,7 +9,7 @@
     "dist"
   ],
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "del-cli -f dist",
```

Comment:
this unrelated change is useful in any case for windows compat

Reply:
If necessary I can create a separate PR.

Reply:
Hey @wardpeet did you hit this in user land or are you trying to make the project more friendly to develop on Widows machines?

Reply:
@jasonkuhrt I'm a windows user so I've hit this issue when doing a `npm install`. If you want to not trigger it on `npm install` you have to use `prepublishOnly` as described in https://docs.npmjs.com/misc/scripts

Reply:
@wardpeet Thanks, remember now. Yep what we want is `prepublishOnly`. Could you per change make that change in this PR, and undo this bit. Cool?